src/shared/ui/toast.tsx

```diff
@@ -0,0 +1,45 @@
+import error from '@/src/assets/icon/medium/error.svg'
+import logo from '@/src/assets/images/(logo)/logo_white.svg'
+import Image from 'next/image'
+
+type ToastProps = {
+  type: 'warning' | 'notice'
+  message: string
+}
+
+const TOAST_STYLES = {
+  warning:
+    'fixed top-[30px] left-1/2 transform -translate-x-1/2 z-[10001] flex flex-row items-center justify-center w-[355px] h-[55px] px-[20px] py-[10px] bg-white rounded-[10px] shadow-custom border-[1.5px] border-error',
+  notice:
+    'fixed top-[30px] left-1/2 transform -translate-x-1/2 z-[10001] flex flex-row items-center w-[355px] h-[55px] px-[20px] py-[10px] bg-brand rounded-[10px] shadow-custom',
+}
+
+const ICONS = {
+  warning: {
+    src: error,
+    alt: 'error',
+    gap: 'gap-[8px]',
```

Comment:
```suggestion
    gap: 'gap-[10px]',
```
디자인 확인해보니 아이콘과 텍스트의 gap이 모든 양식에서 10px이네요!

Reply:
이래서 피그마 프로버전을 써야 하는군요...
<img width="360" alt="image" src="https://github.com/user-attachments/assets/2d5e2592-278a-436e-92bf-afb8e8a9f040" />
프로버전은 텍스트 기준으로 확실히 세부적으로 볼 수 있나봅니다ㅠ

Reply:
제가 잘못 확인했습니다. 
이건 8px이 맞네요!!!